### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by MisterMickey**
Resolve issue #65 

# Changes made : 

- Remove whitespace and merge both divs
- Add title to user id for more clarification
- Add text User ID before actual id
- Change cursor to pointer on User ID
- Change font to Poppins

# Old 

Screenshot from 2023-07-01 12-33-21

# New
Screenshot from 2023-07-01 12-32-14

Screenshot from 2023-07-01 12-32-53

### CODE REVIEW

File: client/src/views/Chat.vue [hunk 1]
```diff
@@ -329,6 +328,17 @@ export default {
...
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*{
+  font-family: "Poppins", "Helvetica", sans-serif;
+}
+
+strong:hover{
+  cursor: pointer;
+}
```
Comment (PradyumnaKrishna) [inline client/src/views/Chat.vue:340]: This hover behaviour is not limited to userId but all bold elements, You can add this to #userId as well.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]
Looks good, I've reviewed this PR. Mostly style related changes required.

File: client/src/views/Chat.vue [hunk 2]
```diff
@@ -329,6 +328,17 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*{
+  font-family: "Poppins", "Helvetica", sans-serif;
+}
```
[the remark this answers is not in the document]
Comment (MisterMickey) [reply client/src/views/Chat.vue:336]: > I guess we don't this font, any specific reason?

This font makes the chat section looks kinda credible and secure and not like some default font.

[the remark this answers is not in the document]
Comment (MisterMickey) [reply client/src/views/Chat.vue:336]: @PradyumnaKrishna If you want I can remove the font..But in my opinion this one is making website more credible.

[the remark this answers is not in the document]
Comment (PradyumnaKrishna) [reply client/src/views/Chat.vue:336]: ok then lets use `Roboto` font.

File: client/src/views/Chat.vue [hunk 3]
```diff
@@ -370,12 +380,25 @@ input::placeholder {
   color: gray;
 }
 
+#userId{
```
[the remark this answers is not in the document]
Comment (MisterMickey) [reply client/src/views/Chat.vue:383]: Done.

File: client/src/views/Chat.vue (hunk 1 above)
Comment (MisterMickey) [reply client/src/views/Chat.vue:340]: Done.

**Comment by MisterMickey** [comment]
@PradyumnaKrishna You can check the PR now.

File: client/src/views/Chat.vue [hunk 4]
```diff
@@ -335,7 +335,7 @@ export default {
   font-family: "Poppins", "Helvetica", sans-serif;
 }
 
-strong:hover{
+#userId:hover{
```
Comment (PradyumnaKrishna) [inline client/src/views/Chat.vue:338]: Actually you can move this to `.inputBox`.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]

**Comment by MisterMickey** [comment]
@PradyumnaKrishna Have a look now.

File: client/src/views/Chat.vue [hunk 5]
```diff
@@ -329,6 +328,17 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*{
+  font-family: "Roboto","Poppins", "Helvetica", sans-serif;
+}
```
Comment (PradyumnaKrishna) [inline client/src/views/Chat.vue:336]:
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');

*{
  font-family: "Roboto", "Helvetica", sans-serif;
}
```

Comment (PradyumnaKrishna) [reply client/src/views/Chat.vue:336]: This is the final change required to get this merged.

**Review by PradyumnaKrishna — CHANGES_REQUESTED** [review]

File: client/src/views/Chat.vue (hunk 5 above)
Comment (MisterMickey) [reply client/src/views/Chat.vue]: @PradyumnaKrishna Done Sir _/\_

**Review by PradyumnaKrishna — APPROVED** [review]

**Comment by PradyumnaKrishna** [comment]
Thank you @MisterMickey for your contributions.

**Pull request merged**